### PR TITLE
Added an explanation on installation steps for Google Colab

### PR DESCRIPTION
**Pull request by SpectrixOfficial**
I had been sharing this program with many people who were unfamiliar with programming and would've liked to use this piece of software, but didn't understand some installation steps. So I decided to make a short-yet-verbose explanation. All this does is explain what the user must do on their side before running. People I have showed this to agreed it helped (various people who were previously confused). You can decide if you'd like to include this, or edit it so it suits your needs.

### CODE REVIEW

**Comment by alievk** [comment]
I like the idea, the device installation step was really easy to miss.
However, why not to use `scripts/install*` for installation instead of manually setting up the environment?

**Comment by bobvo23** [comment]
I agree with alievk. For example, these simple steps requires a fundamental knowledge about conda. Seems that the Macos installation script handles these steps automatically.
   "1.   Open Anaconda Prompt and `cd` into avatarify\n",
    "2.  Create an environment with `conda create --name avatarify`\n",

**Comment by SpectrixOfficial** [comment]
I agree with both of you, I should've thought about that. I removed those two steps.

File: avatarify.ipynb [hunk 1]
```diff
@@ -89,6 +89,26 @@
...
+        "id": "_PpJBfF1lrhw",
+        "colab_type": "text"
+      },
+      "source": [
+        "## Before you start:\n",
+        "Ensure you have the following installed on your device:\n",
+        "\n",
+        "*   [Python 3.7.x](https://www.python.org/downloads/)\n",
+        "*   [Miniconda](https://docs.conda.io/en/latest/miniconda.html#linux-installers)\n",
+        "*   [ngrok](https://ngrok.com/download)\n",
```
Comment (alievk) [inline avatarify.ipynb:104]: You don't need ngrok on the device, only on the server.

**Pull request closed without merging**